### DISCUSSION TITLE
Popup unterdrücken wenn...

### DISCUSSION THREAD

**Discussion by tschortschde** [Zefau/ioBroker.jarvis#2520] [Q&A]
Hallo,

ist es möglich das Popup zu unterdrücken, wenn keine "Fenster" geöffnet sind?
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/b036d9ec-829c-40f0-8b3b-f384f64c3a1d)
Das bloße Anzeigen des Popups ohne Inhalt macht ja keinen Sinn...

Wenn Fester geöffnet sind, soll es natürlich aufgehen... ;-)
![image](https://github.com/Zefau/ioBroker.jarvis/assets/11274233/1d4b5541-1735-45db-a3c6-b792178b318f)

**Comment by tschortschde** [comment]
Gibt es hierzu eine Lösung?

**Comment by mcuiobroker** [comment]
Evtl kannst du es hiermit machen?
```
// Im Body-Stil setzen
{"{0_userdata.0.jarvis.test.num6}=10":{"pointer-events":"none !important"}}
```

**Reply by tschortschde** [reply]
Leider funktioniert es nicht mehr mit der jarvis.0, v3.2.0-rc.12
Was muss ich machen?

**Reply by mcuiobroker** [reply]
Da musst du dann leider warten bis @Zefau Zeit hat sich das anzuschauen. Da kann ich nichts beeinflussen.

**Reply by tschortschde** [reply]
> Da musst du dann leider warten bis @Zefau Zeit hat sich das anzuschauen. Da kann ich nichts beeinflussen.

Es liegt wohl am Umlaut. Wenn ein ä,ü oder ö verwendet wird. Wird die Zeile nicht ausgeblendet. 
Beispiel:
Büro bleibt sichtbar
Wohnzimmer wird ausgeblendet

**Comment by tschortschde** [comment]
Wie cool ist das denn? 
Funktioniert 😀
Danke
